vitess.io/_includes/analytics.liquid

```diff
@@ -0,0 +1,10 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+  if (window.location.hostname == 'youtube.github.io') {
+    ga('create', '{{ site.analytics.id }}', 'auto');
```

Comment:
you didn't set analytics.id in _config.yml. I've enabled google analytics for the site already and you may want to use id "UA-60219601-1" in the config file.